### PR TITLE
implemented backend interaction with s3 bucket

### PR DESCRIPTION
**Pull request by PVigar88**
Co-authored-by: Liliana lilynleyva@gmail.com

in the config s3.js there are a couple functions that do the interactions with the s3 bucket.  This also adds multer as a dependency. 

The imageRouter.js is essentially just there to test the s3 uploads were working.  Talking with Jake, the actual upload will need to take place as part of the submissions record maintenance.  

- What work was done?
Added functions that connect to s3 bucket for upload of files.
- Why was this work done?
to support future development of the record maintenance for submissions
- What feature / user story is it for?
work completed as part of https://trello.com/c/VuvmvlAt/776-create-an-aws-s3-cloud-storage-protocol-for-storing-data

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

### CODE REVIEW

**Comment by hanselviva** [comment]
Not very knowledgeable on S3 bucket but it doesn't look like there's anything that could potentially affect the current state of the BE or anything that could be a problem in the future. Very nice job @PVigar88 and @Liliana-Leyva!

**Review by hanselviva — COMMENTED** [review]
Not very knowledgeable on S3 bucket but it doesn't look like there's anything that could potentially affect the current state of the BE (in a negative way) or anything that could be a problem in the future. Other than linting issues, very nice job @PVigar88 and @Liliana-Leyva!

File: config/aws.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
-/* istanbul ignore file */
+// /* istanbul ignore file */
+// module.exports = {
+//   s3: {
+//     config: {
+//       accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+//       secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+//     },
+//   },
+// };
```
Comment (chrislauyc) [inline config/aws.js]: Commented out code should be removed

File: api/index/indexRouter.js [hunk 2]
```diff
@@ -33,3 +33,5 @@ router.get('/', function (req, res) {
 });
 
 module.exports = router;
+
+
```
Comment (chrislauyc) [inline api/index/indexRouter.js]: This file probably doesn't have to be included in the PR

**Review by chrislauyc — CHANGES_REQUESTED** [review]
Good work implementing the feature!

File: config/aws.js (hunk 1 above)
Comment (PVigar88) [reply config/aws.js]: actually this files shouldn't have been committed either.

File: api/index/indexRouter.js (hunk 2 above)
Comment (PVigar88) [reply api/index/indexRouter.js]: Good Catch, Thanks!

**Review by chrislauyc — APPROVED** [review]

**Review by ashtilawat — CHANGES_REQUESTED** [review]
Could you explain the new package you are adding in your description. Why you need to use it all?

**Pull request closed without merging**